contrib/babelfishpg_common/src/encoding/mb/conv.c

```diff
@@ -401,10 +401,6 @@ TsqlLocalToUtf(const unsigned char *iso, int len,
 		unsigned char b3 = 0;
 		unsigned char b4 = 0;
 
-		/* "break" cases all represent errors */
-		if (*iso == '\0')
-			break;
-
```

Comment:
As discussed last time, we need to handle this differently.

Reply:
Something like this ?